assertk-common/src/test/kotlin/test/assertk/assertJLite.kt

```diff
@@ -0,0 +1,46 @@
+package test.assertk
+
+import assertk.catch
+import kotlin.reflect.KClass
+import kotlin.test.assertEquals
+import kotlin.test.assertTrue
+
+class Assertions {
```

Comment:
I'm not really a fan of this assertjlite class. Is this just a placeholder until the tests themselves are updated to get rid of the assertj dep?

Reply:
Yes, it was just a way to get rid of assertj without changing too much tests.
I think we can inline it within the tests that use it. I just didn't want to change the tests while doing that huge refactoring.

Reply:
Yep, that makes sense